src/Template/Element/Flash/default.ctp

```diff
@@ -1,6 +1,6 @@
 <?php
 $class = array_unique((array)$params['class']);
-$message = h($message);
+$message = isset($params['escape']) && $params['escape'] === false ? $message : h($message);
```

Comment:
Shouldn't the condition check be in `()`? This will fail and return no message if `escape` is not set - https://3v4l.org/R6jU1